setuptools_webpack.py

```diff
@@ -31,4 +32,4 @@ def finalize_options(self):
     def run(self):
         if self.distribution.webpack_output_path:
             output_path = os.path.join(self.build_lib, self.distribution.webpack_output_path)
-            os.system("webpack --output-path '%s'" % output_path)
+            subprocess.check_call("webpack --output-path '%s'" % output_path, shell=True)
```

Comment:
* If the command fails, ti should raise a `DistutilsExecError`.
* I'd avoid using `shell=True`. I don't think it is necessary.